tests/ast_dump_with_exclude_doc.phpt

```diff
@@ -0,0 +1,56 @@
+--TEST--
+ast_dump() with AST_DUMP_EXCLUDE_DOC
+--SKIPIF--
+<?php
+if (!extension_loaded("ast")) print "skip";
+if (PHP_VERSION_ID >= 70000 && PHP_VERSION_ID < 70018) print "skip buggy PHP version";
+if (PHP_VERSION_ID >= 70100 && PHP_VERSION_ID < 70104) print "skip buggy PHP version";
+?>
+--FILE--
+<?php
+
+require __DIR__ . '/../util.php';
+
+$code = <<<'PHP'
+<?php
+/**
+ * test
+ * @param mixed $foo
+ * @return void
+ */
+function
+test
+($foo)
+{
+    var_dump(
+        $foo
+    );
+}
+PHP;
+
+$ast = ast\parse_code($code, $version=50);
```

Comment:
Please use `$version=80` here to make this test longer lift. I don't think it affects the output.

Reply:
AST-version set to 80
3191a4e